diff --git a/src/cpu/s3c2410/gta01.c b/src/cpu/s3c2410/gta01.c
index 4686fa2..a95578b 100644
--- a/src/cpu/s3c2410/gta01.c
+++ b/src/cpu/s3c2410/gta01.c
@@ -266,8 +266,8 @@ const struct board_api board_api_gta01 = {
 			       "loglevel=4 "
 			       "console=tty0 "
 			       "console=ttySAC0,115200 "
-			       "init=/sbin/init "
-			       "ro ",
+			       "init=/usr/bin/bootchart-lite "
+			       "rw ",
 	.commandline_board_debug = " loglevel=8 ",
 	.noboot = "boot/noboot-GTA01",
 	.append = "boot/append-GTA01",
diff --git a/src/cpu/s3c2442/gta02.c b/src/cpu/s3c2442/gta02.c
index 48ee245..5dda349 100644
--- a/src/cpu/s3c2442/gta02.c
+++ b/src/cpu/s3c2442/gta02.c
@@ -693,8 +693,8 @@ const struct board_api board_api_gta02 = {
 	.commandline_board = "loglevel=4 "
 				      "console=tty0 "
 				      "console=ttySAC2,115200 "
-				      "init=/sbin/init "
-				      "ro ",
+				      "init=/usr/bin/bootchart-lite "
+				      "rw ",
 	.commandline_board_debug =  " loglevel=8",
 	.noboot = "boot/noboot-GTA02",
 	.append = "boot/append-GTA02",
diff --git a/src/cpu/s3c6410/om_3d7k-steppingstone.c b/src/cpu/s3c6410/om_3d7k-steppingstone.c
index 71c57f9..62564d0 100644
--- a/src/cpu/s3c6410/om_3d7k-steppingstone.c
+++ b/src/cpu/s3c6410/om_3d7k-steppingstone.c
@@ -87,10 +87,10 @@ const struct board_api board_api_om_3d7k = {
 	.get_ui_keys = get_ui_keys_om_3d7k,
 	.commandline_board = "console=tty0 "
 			     "console=ttySAC3,115200 "
-			     "init=/sbin/init "
+			     "init=/usr/bin/bootchart-lite "
 			     "loglevel=8 "
 			     "rootdelay=1 no_console_suspend "
-			     "ro ",
+			     "rw ",
 	.commandline_board_debug =  " loglevel=8",
 	.kernel_source = {
 		[0] = {
