From af052d388946070a6055d5590bc6e22aa4013139 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Wed, 4 May 2011 22:34:44 +0200
Subject: [PATCH 2/2] create old shr-bindings.h for compatibility with older
 code

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/Makefile.am |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index 5a2b6cf..b79aeb6 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -12,6 +12,7 @@ GDBUSCODEGENFLAGS = \
 BUILT_SOURCES = libshr_glib_la.bindings.stamp
 
 SHRHEADERS = \
+	shr-bindings.h \
 	shr-phonefsousage.h \
 	shr-phoneuicallmanagement.h \
 	shr-phoneuicontacts.h \
@@ -43,12 +44,14 @@ libshr_glibdir = $(includedir)/shr-glib
 libshr_glib_HEADERS = \
 	$(SHRHEADERS)
 
+shr-bindings.h: gen-bindings
+	touch $@
 
 libshr_glib_la.bindings.stamp: gen-bindings $(libshr_glib_la_SOURCES)
 	touch $@
 
 gen-bindings:
-	for f in $(SHR_SPECS_DIR)/*.xml; do gdbus-codegen --interface-prefix org.shr --generate-c-code `echo $$f | sed 's/.*\///g; s/^org\.shr\./shr-/g; s/.xml//g; s/\.//g;' | awk  '{ printf tolower($$1)}'` $$f; done
+	for f in $(SHR_SPECS_DIR)/*.xml; do OUT=`echo $$f | sed 's/.*\///g; s/^org\.shr\./shr-/g; s/.xml//g; s/\.//g;' | awk  '{ printf tolower($$1)}'`; gdbus-codegen  --interface-prefix org.shr --generate-c-code $$OUT $$f; echo "#include <$$OUT.h>" >> shr-bindings.h; done
 
 libshr_glib_la_LIBADD = \
 	$(DEPS_LIBS)
-- 
1.7.5.rc3

