From 9fba492b423c8adcd72727e0b2f67136526d68b8 Mon Sep 17 00:00:00 2001
From: Christophe Chapuis <chris.chapuis@gmail.com>
Date: Fri, 8 Mar 2019 21:41:08 +0000

---
 drivers/video/backlight/lm3630a_bl.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/video/backlight/lm3630a_bl.c b/drivers/video/backlight/lm3630a_bl.c
index 394deb4f5fde..ba24734f4354 100644
--- a/drivers/video/backlight/lm3630a_bl.c
+++ b/drivers/video/backlight/lm3630a_bl.c
@@ -398,11 +398,11 @@ static int lm3630a_probe(struct i2c_client *client,
 			return -ENOMEM;
 		/* default values */
 		pdata->leda_ctrl = LM3630A_LEDA_ENABLE;
-		pdata->ledb_ctrl = LM3630A_LEDB_ENABLE;
+		pdata->ledb_ctrl = LM3630A_LEDB_ON_A;
 		pdata->leda_max_brt = LM3630A_MAX_BRIGHTNESS;
 		pdata->ledb_max_brt = LM3630A_MAX_BRIGHTNESS;
-		pdata->leda_init_brt = 128;
-		pdata->ledb_init_brt = LM3630A_MAX_BRIGHTNESS;
+		pdata->leda_init_brt = 180;
+		pdata->ledb_init_brt = 180;
 	}
 	pchip->pdata = pdata;
 
