From ef9c66468c85e896a938540f697e66d08ac11696 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Sat, 22 Feb 2014 16:31:40 +0100
Subject: [PATCH] illume_kbd: Fix compliation with new efl-1.8

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 7146591..1c05c45 100644
--- a/setup.py
+++ b/setup.py
@@ -63,7 +63,7 @@ setup(
     author_email = "mlauer@vanille-media.de",
     url = "http://www.freesmartphone.org",
     ext_modules = [
-        Extension( "illume", sources=['src/illume.pyx'], **pkgconfig('"ecore >= 0.9.9.050" ''"eina >= 0.0.1"') )
+        Extension( "illume", sources=['src/illume.pyx'], **pkgconfig('"ecore-x >= 0.9.9.050" ''"eina >= 0.0.1"') )
         ],
     cmdclass = { 'build_ext': build_ext ,
                  'build'    : my_build  ,
-- 
1.8.5.3

