From 8bcd30657ca1916e58723961ecb98256255cf41b Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Tue, 11 Sep 2012 13:03:48 +0200
Subject: [PATCH] adapt to API change in r59160 s/homogenous/homogeneous/g

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 ecalc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ecalc b/ecalc
index 8f6b825..83bad59 100755
--- a/ecalc
+++ b/ecalc
@@ -123,7 +123,7 @@ class MainGUI_E:
     tab = e.Table(mainbox)
     tab.size_hint_align_set(-1.0, -1.0)
     tab.size_hint_weight_set(1.0, 1.0)
-    tab.homogenous_set(True)
+    tab.homogeneous_set(True)
     for (i, key) in enumerate(self.keys):
       bt = e.Button(tab)
       bt.size_hint_align_set(-1.0, -1.0)
-- 
1.7.12

