diff -uNr git.orig//configure.ac git/configure.ac
--- git.orig//configure.ac	2011-04-06 00:13:39.565728040 +0200
+++ git/configure.ac	2011-04-06 00:08:31.603727816 +0200
@@ -7,7 +7,7 @@
 m4_pattern_allow
 AM_PROG_VALAC([0.11.4])
 
-PKG_CHECK_MODULES([DEPS], [glib-2.0 gio-2.0 elementary gee-1.0])
+PKG_CHECK_MODULES([DEPS], [glib-2.0 gio-2.0 evas elementary gee-1.0])
 
 AC_CHECK_LIB(gthread-2.0, g_thread_init)
 AC_CHECK_LIB(libeflvala)
