diff --git a/src/Makefile.am b/src/Makefile.am
index 70bf8c9..831233a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -1,5 +1,5 @@
 AM_VALAFLAGS =  \
-	--pkg "gio-2.0"  --pkg "glib-2.0" --pkg "elm" --pkg "gee-1.0" \
+	--pkg "gio-2.0"  --pkg "glib-2.0" --pkg "elementary" --pkg "gee-1.0" \
 	--thread
 
 AM_CFLAGS = $(DEPS_CFLAGS)
diff --git a/src/view_main.vala b/src/view_main.vala
index 57e503d..8840665 100644
--- a/src/view_main.vala
+++ b/src/view_main.vala
@@ -43,13 +43,13 @@ namespace iliwi.View {
     //itc.item_style = "double_label";
     itc.item_style = "default";
     itc.func.label_get = genlist_get_label;
-    itc.func.icon_get = null;
+    itc.func.content_get = null;
     itc.func.state_get = null;
     itc.func.del = null;
 
     itc2.item_style = "default";
     itc2.func.label_get = certlist_get_label;
-    itc2.func.icon_get = null;
+    itc2.func.content_get = null;
     itc2.func.state_get = null;
     itc2.func.del = null;
     
@@ -143,7 +143,7 @@ namespace iliwi.View {
     foreach(Network network in wifi.get_visible_networks()) {
       // Find place (sorted by preferred > strength
       if( items_in_list == false )
-        network.listitem = wifilist.item_append( itc, (void*)network, null, Elm.GenlistItemFlags.NONE, item_select );
+        network.listitem = wifilist.item_append( ref itc, (void*)network, null, Elm.GenlistItemFlags.NONE, item_select );
       else {
         listitem_tmp = wifilist.first_item_get();
         Network list_network = (Network) listitem_tmp.data_get();
@@ -151,16 +151,16 @@ namespace iliwi.View {
         while(found_place == false) {
           if( network.preferred_network && list_network.preferred_network==false ) {
             found_place = true;
-            network.listitem = wifilist.item_insert_before( itc, (void*)network, null, listitem_tmp, Elm.GenlistItemFlags.NONE, item_select );
+            network.listitem = wifilist.item_insert_before( ref itc, (void*)network, null, listitem_tmp, Elm.GenlistItemFlags.NONE, item_select );
           } else if( list_network.preferred_network==network.preferred_network && list_network.strength<=network.strength ) {
             found_place = true;
-            network.listitem = wifilist.item_insert_before( itc, (void*)network, null, listitem_tmp, Elm.GenlistItemFlags.NONE, item_select );
+            network.listitem = wifilist.item_insert_before( ref itc, (void*)network, null, listitem_tmp, Elm.GenlistItemFlags.NONE, item_select );
           } else { // Couldn't find a place to put it
             listitem_tmp2 = listitem_tmp.next_get();
             listitem_tmp = listitem_tmp2;
             if( listitem_tmp==null ) {
               found_place = true;
-              network.listitem = wifilist.item_append( itc, (void*)network, null, Elm.GenlistItemFlags.NONE, item_select );
+              network.listitem = wifilist.item_append( ref itc, (void*)network, null, Elm.GenlistItemFlags.NONE, item_select );
             } else
               list_network = (Network) listitem_tmp.data_get();
           }
@@ -396,7 +396,7 @@ namespace iliwi.View {
     certlist.size_hint_align_set(-1, -1);
     list_cert_dir();
     foreach (Certificate cert in ls) {
-      cert.listitem = certlist.item_append(itc2, (void*)cert, null, Elm.GenlistItemFlags.NONE, cert_select);
+      cert.listitem = certlist.item_append(ref itc2, (void*)cert, null, Elm.GenlistItemFlags.NONE, cert_select);
     }
     certlist.show();
     cert_chooser_page.pack_end(certlist);
