From 5ed3104facbe2281e41487ffd93e056b89ca7994 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Sat, 17 Dec 2011 02:24:51 +0100
Subject: [PATCH] wizard: disable appshadow move to volatile

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 python/shr-wizard.py |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/python/shr-wizard.py b/python/shr-wizard.py
index 6f7c870..7203c6c 100755
--- a/python/shr-wizard.py
+++ b/python/shr-wizard.py
@@ -159,9 +159,9 @@ pager.content_push(wel)
 os.system( "mkdir -p ~/.e/e/applications/startup/" )
 os.system( "echo shr_elm_softkey.desktop > ~/.e/e/applications/startup/.order" )
 os.system( "shr_elm_softkey >/dev/null </dev/null &" )
-os.system( "mv ~/.e/e/appshadow/* /var/volatile/appshadow/" )
-os.system( "rm -rf ~/.e/e/appshadow/" )
-os.system( "ln -s /var/volatile/appshadow/ ~/.e/e/appshadow" )
+#os.system( "mv ~/.e/e/appshadow/* /var/volatile/appshadow/" )
+#os.system( "rm -rf ~/.e/e/appshadow/" )
+#os.system( "ln -s /var/volatile/appshadow/ ~/.e/e/appshadow" )
 
 elementary.run()
 elementary.shutdown()
-- 
1.7.8

