From 57b422f8e50528070d216b2fa8c442dcf9c6d04b Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Tue, 6 Dec 2011 18:54:50 +0100
Subject: [PATCH] menu: fix s/homogenous/homogeneous/g

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/menu.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/menu.c b/src/menu.c
index 5d430ca..cd9ac04 100644
--- a/src/menu.c
+++ b/src/menu.c
@@ -90,7 +90,7 @@ static void create_win(void)
 
     // buttons box
     Evas_Object* btn_box = elm_box_add(win->win);
-    elm_box_homogenous_set(btn_box, TRUE);
+    elm_box_homogeneous_set(btn_box, TRUE);
     evas_object_size_hint_weight_set(btn_box, EVAS_HINT_EXPAND, 0.8);
     evas_object_size_hint_align_set(btn_box, EVAS_HINT_FILL, -1.0);
     evas_object_show(btn_box);
-- 
1.7.8

