From 1d654e54ded4b0e334051f1e2bb87256cde3d32f Mon Sep 17 00:00:00 2001
From: kszaq <kszaquitto@gmail.com>
Date: Sun, 17 Feb 2019 23:39:01 +0100
Subject: [PATCH] hybris: gles: Include GLES3 headers

---
 hybris/include/Makefile.am | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/hybris/include/Makefile.am b/hybris/include/Makefile.am
index 4b5bf397..90daa415 100644
--- a/hybris/include/Makefile.am
+++ b/hybris/include/Makefile.am
@@ -18,6 +18,14 @@ gles2include_HEADERS = \
 	GLES2/gl2ext.h \
 	GLES2/gl2platform.h
 
+gles3includedir = $(includedir)/GLES3
+gles3include_HEADERS = \
+	GLES3/gl3.h \
+	GLES3/gl31.h \
+	GLES3/gl32.h \
+	GLES3/gl3ext.h \
+	GLES3/gl3platform.h
+
 khrincludedir = $(includedir)/KHR
 khrinclude_HEADERS = \
 	KHR/khrplatform.h
