From 18aca0119c0b422450d64eec7d4d434c5c8785a3 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Sat, 17 Dec 2011 03:09:35 +0100
Subject: [PATCH] fix changed callback name

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/gui.c  |   16 ++++++++--------
 src/main.c |    2 +-
 2 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/src/gui.c b/src/gui.c
index e643ccd..3d46367 100644
--- a/src/gui.c
+++ b/src/gui.c
@@ -45,7 +45,7 @@ void popup_message(char *message, char *msg, int butt_state)
 	
 	pop_win = elm_win_add(NULL, "inwin", ELM_WIN_DIALOG_BASIC);
 	elm_win_title_set(pop_win, "Message");
-	evas_object_smart_callback_add(pop_win, "delete-request", pop_win_del, NULL);
+	evas_object_smart_callback_add(pop_win, "delete,request", pop_win_del, NULL);
 
 	bg = elm_bg_add(pop_win);
 	elm_win_resize_object_add(pop_win, bg);
@@ -298,7 +298,7 @@ void create_add_alb_page(void *data, Evas_Object *obj, void *event_info)
 	evas_object_smart_callback_add(cancel_button, "clicked", cancel_button_clicked, &wl);
 	
 	//set delete callback for this window
-	evas_object_smart_callback_add(add_alb_win, "delete-request", cancel_button_clicked, &wl);
+	evas_object_smart_callback_add(add_alb_win, "delete,request", cancel_button_clicked, &wl);
 
 	update_album_list(&itc_add_album_list, add_album_list);
 	evas_object_resize(add_alb_win, 480, 600);	
@@ -817,7 +817,7 @@ void create_dir_browser_page(void *data, Evas_Object *obj, void *event_info)
 	evas_object_smart_callback_add(cancel_button, "clicked", cancel_button_clicked, &wl);
 	
 	//set delete callback for this window
-	evas_object_smart_callback_add(dir_browser_page_win, "delete-request", cancel_button_clicked, &wl);
+	evas_object_smart_callback_add(dir_browser_page_win, "delete,request", cancel_button_clicked, &wl);
 
 	//load directory data
 	load_dir(dir_list, "/");
@@ -957,7 +957,7 @@ void select_songs_page(void *data, Evas_Object *obj, void *event_info)
 	evas_object_smart_callback_add(cancel_button, "clicked", cancel_button_clicked, &wl);
 
 	//set delete callback for this window
-	evas_object_smart_callback_add(select_songs_page_win, "delete-request", cancel_button_clicked, &wl);
+	evas_object_smart_callback_add(select_songs_page_win, "delete,request", cancel_button_clicked, &wl);
 	populate_filter_final_list(cl->type, name, filter_final_list);
 	
 	evas_object_resize(select_songs_page_win, 480, 600);
@@ -1071,7 +1071,7 @@ void sel_songs_by_page(void *data, Evas_Object *obj, void *event_info)
 	evas_object_smart_callback_add(cancel_button, "clicked", cancel_button_clicked, &wl);
 
 	//set delete callback for this window
-	evas_object_smart_callback_add(sel_songs_win, "delete-request", cancel_button_clicked, &wl);
+	evas_object_smart_callback_add(sel_songs_win, "delete,request", cancel_button_clicked, &wl);
 
 	populate_filter_list(type, filter_list);
 	evas_object_resize(sel_songs_win, 480, 600);
@@ -1249,7 +1249,7 @@ void manage_songs_page(void *data, Evas_Object *obj, void *event_info)
 	evas_object_smart_callback_add(cancel_button, "clicked", cancel_button_clicked, &wl);
 	
 	//set delete callback for this window
-	evas_object_smart_callback_add(manage_songs_page_win, "delete-request", cancel_button_clicked, &wl);
+	evas_object_smart_callback_add(manage_songs_page_win, "delete,request", cancel_button_clicked, &wl);
 	
 	evas_object_resize(manage_songs_page_win, 480, 600);
 	evas_object_show(manage_songs_page_win);
@@ -1397,7 +1397,7 @@ void create_manage_playlist_page(void)
 	evas_object_smart_callback_add(cancel_button, "clicked", cancel_button_clicked, &wl);
 	
 	//set delete callback for this window
-	evas_object_smart_callback_add(manage_playlist_page_win, "delete-request", cancel_button_clicked, &wl);
+	evas_object_smart_callback_add(manage_playlist_page_win, "delete,request", cancel_button_clicked, &wl);
 
 	evas_object_resize(manage_playlist_page_win, 480, 600);
 	evas_object_show(manage_playlist_page_win);
@@ -1575,7 +1575,7 @@ void create_playlist_page(void *data, Evas_Object *obj, void *event_info)
 	evas_object_smart_callback_add(cancel_button, "clicked", cancel_button_clicked, &wl);
 	
 	//set delete callback for this window
-	evas_object_smart_callback_add(playlist_page_win, "delete-request", cancel_button_clicked, &wl);
+	evas_object_smart_callback_add(playlist_page_win, "delete,request", cancel_button_clicked, &wl);
 
 	if (strcmp(pl_name, "") == 0) {
 		elm_entry_entry_set(entry, "myplaylist");
diff --git a/src/main.c b/src/main.c
index a38e97e..cdb47a3 100644
--- a/src/main.c
+++ b/src/main.c
@@ -125,7 +125,7 @@ elm_main(int argc, char **argv)
 	else if (strcmp(cur_playlist, "0 All Songs") == 0) strcpy(buf, "All Songs");
 	else strcpy(buf, cur_playlist);
 	elm_win_title_set(win, buf);
-	evas_object_smart_callback_add(win, "delete-request", my_win_del, NULL);
+	evas_object_smart_callback_add(win, "delete,request", my_win_del, NULL);
 	//show the window
 	create_gui(win);
 	evas_object_show(win);
-- 
1.7.8

