From 2944979b0ee6339856e446dbf22a6a5cb29a10a8 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Thu, 8 Mar 2012 16:59:45 +0100
Subject: [PATCH] genlist: adapt to elementary changes from r68943

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/callbacks.c |   12 ++++++------
 src/db_sqlite.c |    4 ++--
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/src/callbacks.c b/src/callbacks.c
index b6eb97b..23cfc6c 100644
--- a/src/callbacks.c
+++ b/src/callbacks.c
@@ -153,7 +153,7 @@ Eina_Bool timer_func(void *data)
 				song_index = 0; seek = 0;
 				if (view == LISTVIEW) {
 					elm_genlist_item_selected_set(song_list[song_index], 1);
-					elm_genlist_item_show(song_list[song_index]);
+					elm_genlist_item_show(song_list[song_index], ELM_GENLIST_ITEM_SCROLLTO_IN);
 				}
 				else if (view == ALBUM_ART) show_song_info();
 				else if (view == LYRICS) show_song_lyrics();
@@ -425,14 +425,14 @@ void prev_button_clicked(void *data, Evas_Object *obj, void *event_info)
 				elm_genlist_item_selected_set(song_list[song_index], 1);
 				elm_genlist_item_update(song_list[song_index]);
 				old_song = song_list[song_index];
-				elm_genlist_item_show(song_list[song_index]);
+				elm_genlist_item_show(song_list[song_index], ELM_GENLIST_ITEM_SCROLLTO_IN);
 			}
 			else {//if randomize
 				elm_genlist_item_selected_set(song_list[seq[song_index+1]], 0);
 				elm_genlist_item_selected_set(song_list[seq[song_index]], 1);
 				elm_genlist_item_update(song_list[seq[song_index]]);
 				old_song = song_list[seq[song_index]];
-				elm_genlist_item_show(song_list[seq[song_index]]);				
+				elm_genlist_item_show(song_list[seq[song_index]], ELM_GENLIST_ITEM_SCROLLTO_IN);				
 			}
 		}
 		else if (view == ALBUM_ART) show_song_info();
@@ -495,7 +495,7 @@ void next_button_clicked(void *data, Evas_Object *obj, void *event_info)
 			elm_genlist_item_selected_set(song_list[song_index], 1);
 			elm_genlist_item_update(song_list[song_index]);
 			old_song = song_list[song_index];
-			elm_genlist_item_show(song_list[song_index]);
+			elm_genlist_item_show(song_list[song_index], ELM_GENLIST_ITEM_SCROLLTO_IN);
 		}
 		else { // if randomize
 			if(song_index > 0) elm_genlist_item_selected_set(song_list[seq[song_index -1]], 0);
@@ -503,7 +503,7 @@ void next_button_clicked(void *data, Evas_Object *obj, void *event_info)
 			elm_genlist_item_selected_set(song_list[seq[song_index]], 1);
 			elm_genlist_item_update(song_list[seq[song_index]]);
 			old_song = song_list[seq[song_index]];
-			elm_genlist_item_show(song_list[seq[song_index]]);
+			elm_genlist_item_show(song_list[seq[song_index]], ELM_GENLIST_ITEM_SCROLLTO_IN);
 		}
 	}
 	else if (view == ALBUM_ART) show_song_info();
@@ -620,7 +620,7 @@ void play_button_clicked(void *data, Evas_Object *obj, void *event_info)
 		if (!ret) return;
 		if (view == LISTVIEW) {
 			elm_genlist_item_update(song_list[song_index]);
-			elm_genlist_item_show(song_list[song_index]);
+			elm_genlist_item_show(song_list[song_index], ELM_GENLIST_ITEM_SCROLLTO_IN);
 			old_song = song_list[song_index];
 		}
 		else if(view == ALBUM_ART) show_song_info();
diff --git a/src/db_sqlite.c b/src/db_sqlite.c
index 20b5966..cb43260 100644
--- a/src/db_sqlite.c
+++ b/src/db_sqlite.c
@@ -723,7 +723,7 @@ void populate_play_list(Evas_Object *li)
 		gl_item = elm_genlist_first_item_get(li);
 	}
 	elm_genlist_item_selected_set(gl_item ,1);
-	elm_genlist_item_show(gl_item);
+	elm_genlist_item_show(gl_item, ELM_GENLIST_ITEM_SCROLLTO_IN);
 }
 
 void populate_playlist_songs(Evas_Object *li)
@@ -765,7 +765,7 @@ void sel_song(void)
 		if (song_to_play) {
 			elm_genlist_item_selected_set(song_to_play, 1);
 			old_song = song_to_play;
-			elm_genlist_item_show(song_to_play);
+			elm_genlist_item_show(song_to_play, ELM_GENLIST_ITEM_SCROLLTO_IN);
 		}
 	}
 }
-- 
1.7.8.5

