From 6ff33eaae17f7aa4b8b5076bf1d45bcb0da73bc9 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Tue, 6 Dec 2011 18:41:48 +0100
Subject: [PATCH 2/3] genlist API change label_get/text_get

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/gui.c |   34 +++++++++++++++++-----------------
 1 files changed, 17 insertions(+), 17 deletions(-)

diff --git a/src/gui.c b/src/gui.c
index db46741..e643ccd 100644
--- a/src/gui.c
+++ b/src/gui.c
@@ -175,7 +175,7 @@ void cancel_button_clicked(void *data, Evas_Object *obj, void *event_info)
 	evas_object_del(wl->win);
 }
 
-char *album_l_label_get(const void *data, Evas_Object *obj, const char *part)
+char *album_l_text_get(const void *data, Evas_Object *obj, const char *part)
 {
 	char tystr[255], *ty;
 	char *label = (char *)data;
@@ -264,7 +264,7 @@ void create_add_alb_page(void *data, Evas_Object *obj, void *event_info)
 
 	//genlist class defs
 	itc_add_album_list.item_style     		= "default";
-	itc_add_album_list.func.label_get 	= album_l_label_get;
+	itc_add_album_list.func.text_get 	= album_l_text_get;
 	itc_add_album_list.func.content_get  	= album_l_content_get;
 	itc_add_album_list.func.state_get 	= album_l_state_get;
 	itc_add_album_list.func.del       		= album_l_del;
@@ -313,7 +313,7 @@ my_hover_bt_1(void *data, Evas_Object *obj, void *event_info)
    evas_object_show(hv);
 }
 
-char *gl_label_get(const void *data, Evas_Object *obj, const char *part)
+char *gl_text_get(const void *data, Evas_Object *obj, const char *part)
 {
 	char tystr[255], *ty;
 	
@@ -517,7 +517,7 @@ void create_gui(Evas_Object *win)
 	
 	//genlist class defs
 	itc_list.item_style     		= "default";
-	itc_list.func.label_get 	= gl_label_get;
+	itc_list.func.text_get 	= gl_text_get;
 	itc_list.func.content_get  	= gl_content_get;
 	itc_list.func.state_get 	= gl_state_get;
 	itc_list.func.del       		= gl_del;
@@ -635,7 +635,7 @@ void create_gui(Evas_Object *win)
 	populate_main_list(list);
 }
 
-char *album_gl_label_get(const void *data, Evas_Object *obj, const char *part)
+char *album_gl_text_get(const void *data, Evas_Object *obj, const char *part)
 {
 	char tystr[255], *ty;
 	char *label = (char *)data;
@@ -668,7 +668,7 @@ void album_gl_del(const void *data, Evas_Object *obj)
 {
 }
 
-char *dir_gl_label_get(const void *data, Evas_Object *obj, const char *part)
+char *dir_gl_text_get(const void *data, Evas_Object *obj, const char *part)
 {
 	_dir *dir_gl = (_dir *)data;
 	char tystr[255], *ty;
@@ -783,7 +783,7 @@ void create_dir_browser_page(void *data, Evas_Object *obj, void *event_info)
 
 	//genlist class defs
 	itc_dir_list.item_style     		= "default";
-	itc_dir_list.func.label_get 		= dir_gl_label_get;
+	itc_dir_list.func.text_get 		= dir_gl_text_get;
 	itc_dir_list.func.content_get  		= dir_gl_content_get;
 	itc_dir_list.func.state_get 	= dir_gl_state_get;
 	itc_dir_list.func.del       		= dir_gl_del;
@@ -838,7 +838,7 @@ void select_all_songs(void *data, Evas_Object *obj, void *event_info)
 	}
 }
 
-char *final_filter_gl_label_get(const void *data, Evas_Object *obj, const char *part)
+char *final_filter_gl_text_get(const void *data, Evas_Object *obj, const char *part)
 {
 	char tystr[255], *ty;
 	_str2 *filter_song = (_str2 *) data;
@@ -914,7 +914,7 @@ void select_songs_page(void *data, Evas_Object *obj, void *event_info)
 
 	//genlist class defs
 	itc_filter_final.item_style     		= "default";
-	itc_filter_final.func.label_get 	= final_filter_gl_label_get;
+	itc_filter_final.func.text_get 	= final_filter_gl_text_get;
 	itc_filter_final.func.content_get  	= final_filter_gl_content_get;
 	itc_filter_final.func.state_get 	= final_filter_gl_state_get;
 	itc_filter_final.func.del       		= final_filter_gl_del;
@@ -964,7 +964,7 @@ void select_songs_page(void *data, Evas_Object *obj, void *event_info)
 	evas_object_show(select_songs_page_win);
 }
 
-char *filter_gl_label_get(const void *data, Evas_Object *obj, const char *part)
+char *filter_gl_text_get(const void *data, Evas_Object *obj, const char *part)
 {
 	char tystr[255], *ty;
 	char *label = (char *)data;
@@ -1035,7 +1035,7 @@ void sel_songs_by_page(void *data, Evas_Object *obj, void *event_info)
 
 	//genlist class defs
 	itc_filter_list.item_style     	= "default";
-	itc_filter_list.func.label_get 	= filter_gl_label_get;
+	itc_filter_list.func.text_get 	= filter_gl_text_get;
 	itc_filter_list.func.content_get  	= filter_gl_content_get;
 	itc_filter_list.func.state_get 	= filter_gl_state_get;
 	itc_filter_list.func.del       		= filter_gl_del;
@@ -1092,7 +1092,7 @@ void play_all_songs(void *data, Evas_Object *obj, void *event_info)
 	elm_win_title_set(win, "All Songs");
 }
 
-char *home_list_label_get(const void *data, Evas_Object *obj, const char *part)
+char *home_list_text_get(const void *data, Evas_Object *obj, const char *part)
 {
 	char tystr[255], *ty;
 	char *label = (char *)data;
@@ -1160,7 +1160,7 @@ void manage_songs_page(void *data, Evas_Object *obj, void *event_info)
 
 	//genlist class defs
 	itc_home_list.item_style     		= "default";
-	itc_home_list.func.label_get 	= home_list_label_get;
+	itc_home_list.func.text_get 	= home_list_text_get;
 	itc_home_list.func.content_get  	= home_list_content_get;
 	itc_home_list.func.state_get 	= home_list_state_get;
 	itc_home_list.func.del       		= home_list_del;
@@ -1255,7 +1255,7 @@ void manage_songs_page(void *data, Evas_Object *obj, void *event_info)
 	evas_object_show(manage_songs_page_win);
 }
 
-char *play_gl_label_get(const void *data, Evas_Object *obj, const char *part)
+char *play_gl_text_get(const void *data, Evas_Object *obj, const char *part)
 {
 	char tystr[255], *ty;
 	char *label = (char *)data;
@@ -1335,7 +1335,7 @@ void create_manage_playlist_page(void)
 
 	//genlist class defs
 	itc_play_list.item_style     		= "default";
-	itc_play_list.func.label_get 	= play_gl_label_get;
+	itc_play_list.func.text_get 	= play_gl_text_get;
 	itc_play_list.func.content_get  	= play_gl_content_get;
 	itc_play_list.func.state_get 	= play_gl_state_get;
 	itc_play_list.func.del       		= play_gl_del;
@@ -1406,7 +1406,7 @@ void create_manage_playlist_page(void)
 	populate_play_list(play_list);
 }
 
-char *c_play_gl_label_get(const void *data, Evas_Object *obj, const char *part)
+char *c_play_gl_text_get(const void *data, Evas_Object *obj, const char *part)
 {
 	char tystr[255], *ty;
 	char *label = (char *)data;
@@ -1531,7 +1531,7 @@ void create_playlist_page(void *data, Evas_Object *obj, void *event_info)
 
 	//genlist class defs
 	itc_c_play_list.item_style     		= "default";
-	itc_c_play_list.func.label_get 	= c_play_gl_label_get;
+	itc_c_play_list.func.text_get 	= c_play_gl_text_get;
 	itc_c_play_list.func.content_get  	= c_play_gl_content_get;
 	itc_c_play_list.func.state_get 	= c_play_gl_state_get;
 	itc_c_play_list.func.del       		= c_play_gl_del;
-- 
1.7.8.2

