Index: a/chroneo.py
===================================================================
--- a/chroneo.py	(revision 9)
+++ a/chroneo.py	(working copy)
@@ -138,7 +138,7 @@
         # buttons
         box_buttons = elementary.Box(self.main.win)
         box_buttons.horizontal_set(True)
-        box_buttons.homogenous_set(True)
+        box_buttons.homogeneous_set(True)
         box_buttons.size_hint_align_set(-1.0, 0)
         self.box.pack_end(box_buttons)
         box_buttons.show()
@@ -274,7 +274,7 @@
         # buttons
         box_buttons = elementary.Box(self.main.win)
         box_buttons.horizontal_set(True)
-        box_buttons.homogenous_set(True)
+        box_buttons.homogeneous_set(True)
         box_buttons.size_hint_align_set(-1.0, 0)
         self.box.pack_end(box_buttons)
         box_buttons.show()
@@ -407,8 +407,8 @@
         toggle.style_set("toggle")
         toggle.size_hint_align_set(-1, 0)
         toggle.state_set(db.get_setting('fullscreen') or False)
-        toggle.text_part_set("on", "Yes")
-        toggle.text_part_set("off", "No")
+        toggle.part_text_set("on", "Yes")
+        toggle.part_text_set("off", "No")
         toggle.callback_changed_add(self.toggle_fullscreen)
         box.pack_end(toggle)
         toggle.show()
@@ -419,8 +419,8 @@
         toggle.style_set("toggle")
         toggle.size_hint_align_set(-1, 0)
         toggle.state_set(db.get_setting('rotation') or False)
-        toggle.text_part_set("off", "Portrait")          
-        toggle.text_part_set("on", "Landscape")
+        toggle.part_text_set("off", "Portrait")          
+        toggle.part_text_set("on", "Landscape")
         toggle.callback_changed_add(self.toggle_rotation)
         box.pack_end(toggle)
         toggle.show()
@@ -431,8 +431,8 @@
         toggle.style_set("toggle")
         toggle.size_hint_align_set(-1, 0)
         toggle.state_set(db.get_setting('sounds') or False)
-        toggle.text_part_set("on", "Yes")                      
-        toggle.text_part_set("off", "No") 
+        toggle.part_text_set("on", "Yes")                      
+        toggle.part_text_set("off", "No") 
         toggle.callback_changed_add(self.toggle_sounds)
         box.pack_end(toggle)
         toggle.show()
@@ -542,13 +542,14 @@
         self.pcm.setchannels(1)
         self.pcm.setrate(22050)
         self.pcm.setformat(alsaaudio.PCM_FORMAT_U8)
-        self.mixer = alsaaudio.Mixer('PCM', cardindex = 0)
+        self.mixer = alsaaudio.Mixer(control='Media', cardindex = 0)
 
+
         self.win = elementary.Window(APP_NAME, elementary.ELM_WIN_BASIC)
         self.win.title_set(APP_NAME)
         if db.get_setting('rotation') is not None:
             self.win.rotation_set(270 * int(db.get_setting('rotation')))
-        self.win.callback_destroy_add(self.quit)
+        self.win.callback_delete_request_add(self.quit)
 
         bg = elementary.Background(self.win)
         self.win.resize_object_add(bg)
@@ -562,7 +563,7 @@
 
         toolbar = elementary.Toolbar(self.win)
         toolbar.menu_parent_set(self.win)
-        toolbar.homogenous_set(False)
+        toolbar.homogeneous_set(False)
         toolbar.size_hint_align_set(-1.0, 0)
         box.pack_end(toolbar)
         toolbar.show()
