From e6fffa7bb3abd09bf02305ab7f4f9e96d12ee2d1 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Fri, 2 Dec 2011 01:45:08 +0100
Subject: [PATCH] fix name of changed genlist/gengrid callbacks

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 ffalarms.vala |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/ffalarms.vala b/ffalarms.vala
index bed1c65..710e9b5 100644
--- a/ffalarms.vala
+++ b/ffalarms.vala
@@ -1445,7 +1445,7 @@ class Alarms
     {
 	lst = new Genlist(parent);
 	itc.item_style = "double_label";
-	itc.func.label_get = (GenlistItemLabelGetFunc) get_label;
+	itc.func.text_get = (GenlistItemTextGetFunc) get_text;
     }
 
     public void update(Config cfg) throws MyError
@@ -1470,7 +1470,7 @@ class Alarms
 	return (item != null) ? ((NextAlarm) item.data_get()).comp : null;
     }
 
-    static string get_label(void *data, Elm.Object? obj, string part)
+    static string get_text(void *data, Elm.Object? obj, string part)
     {
 	return ((NextAlarm) data).get_label(part);
     }
-- 
1.7.8.rc4

