diff --git a/arch/arm/kernel/sysfs_v7.c b/arch/arm/kernel/sysfs_v7.c
index 5ed32fb..96a3c16 100644
--- a/arch/arm/kernel/sysfs_v7.c
+++ b/arch/arm/kernel/sysfs_v7.c
@@ -76,7 +76,8 @@ static ssize_t aux_ctl_store(struct sys_device *dev,
 	asm ("mrc p15, 0, %0, c1, c0, 1" : "=r"(val));
 	SETBITS(val, 0xff8, new);
 	val &= ~2;
-	asm ("mov r0,  %0	\n\t"
+	asm (".arch_extension sec\n\t"
+	     "mov r0,  %0	\n\t"
 	     "mov r12, #3	\n\t"
 	     "smc #0		\n\t"
 	     :: "r"(val) : "r0", "r12");
@@ -107,7 +108,8 @@ static ssize_t l2_aux_ctl_store(struct sys_device *dev,
 
 	asm ("mrc p15, 1, %0, c9, c0, 2" : "=r"(val));
 	SETBITS(val, 0xbc00000, new);
-	asm ("mov r0,  %0	\n\t"
+	asm (".arch_extension sec\n\t"
+	     "mov r0,  %0	\n\t"
 	     "mov r12, #2	\n\t"
 	     "smc #0		\n\t"
 	     :: "r"(val) : "r0", "r12");
