--- ace-1.4/lib/Makefile.am	2012-12-22 16:25:30.010194270 +0100
+++ ace-1.4.new/lib/Makefile.am	2012-12-22 16:32:16.216168795 +0100
@@ -24,7 +24,7 @@
 	./make-imglib -n cards -i $(srcdir)/png -d images.d $(srcdir)/images.in images.c
 
 make-imglib : make-imglib.c
-	$(BUILD_CC) $(CFLAGS) $(CPPFLAGS) $(LDFLAGS) $(srcdir)/make-imglib.c -o make-imglib -lpng -lz
+	$(BUILD_CC) $(BUILD_CFLAGS) $(BUILD_CPPFLAGS) $(BUILD_LDFLAGS) $(srcdir)/make-imglib.c -o make-imglib -lpng -lz
 
 text2c : text2c.c
-	$(BUILD_CC) $(CFLAGS) $(CPPFLAGS) $(LDFLAGS) $(srcdir)/text2c.c -o text2c -lpng -lz
+	$(BUILD_CC) $(BUILD_CFLAGS) $(BUILD_CPPFLAGS) $(BUILD_LDFLAGS) $(srcdir)/text2c.c -o text2c -lpng -lz
