From b7cfb2a21fcb73daf6497f2f3a0395d71b91def0 Mon Sep 17 00:00:00 2001
From: Christophe Chapuis <chris.chapuis@gmail.com>
Date: Sat, 13 Apr 2019 08:10:13 +0000
Subject: [PATCH] tenderloin: Fix userdata mount options

Signed-off-by: Christophe Chapuis <chris.chapuis@gmail.com>
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 scripts/halium | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/halium b/scripts/halium
index 9d8cdc4..1282fdd 100644
--- a/scripts/halium
+++ b/scripts/halium
@@ -447,7 +447,7 @@ mountroot() {
 
 	# Mount the data partition to a temporary mount point
 	# FIXME: data=journal used as a workaround for bug 1387214
-	mount -o discard,data=journal $path /tmpmnt
+	mount  $path /tmpmnt
 
 	# Set $_syspart if it is specified as systempart= on the command line
 	if grep -q systempart= /proc/cmdline; then
