=== modified file 'src/gui/estardict-gui-dictionariesPage.c'
--- estardict-0.5.orig/src/gui/estardict-gui-dictionariesPage.c	2012-01-02 10:20:47 +0000
+++ estardict-0.5/src/gui/estardict-gui-dictionariesPage.c	2012-02-09 12:21:14 +0000
@@ -41,13 +41,13 @@
     g_debug("-> %s %s()", __FILE__, __FUNCTION__);
 
     Evas_Object *genList = data;
-    Elm_Genlist_Item *gli = elm_genlist_selected_item_get(genList);
+    Elm_Object_Item *gli = elm_genlist_selected_item_get(genList);
     if (!gli) {
         g_message("Item not selected.");
         return;
     }
 
-    gchar* dictionaryToRemove = (gchar *)elm_genlist_item_data_get(gli);
+    gchar* dictionaryToRemove = (gchar *)elm_object_item_data_get(gli);
 
     // Remove from dictionary list
     const Eina_List *l;
@@ -60,7 +60,7 @@
     }
 
     // Remove from genList
-    elm_genlist_item_del(gli);
+    elm_object_item_del(gli);
 }
 
 
@@ -237,7 +237,7 @@
  */
 void clearDictionaryList() {
   g_debug("-> %s %s()", __FILE__, __FUNCTION__);
-  elm_gen_clear(genList);
+  elm_genlist_clear(genList);
   dictionariesList = eina_list_free(dictionariesList);
 }
 
@@ -254,7 +254,7 @@
     DictionaryProperties *dictionary;
 
     // If genList is empty
-    if ( elm_gen_first_item_get(genList) == NULL ) {
+    if ( elm_genlist_first_item_get(genList) == NULL ) {
         EINA_LIST_FOREACH(dictionaries, l, dictionary) {
             if ( dictionary != NULL && 
                 dictionary->dictionaryPath != NULL && 
@@ -273,7 +273,7 @@
  * @param selected Selected dictionary file
  */
 void addDictionaryToGenlist(const gchar *selected) {
-    Elm_Genlist_Item *gli;
+    Elm_Object_Item *gli;
 
     if ( selected != NULL )  {
         itc.item_style = "default";

