Index: podboy/gui.py
===================================================================
--- podboy/gui.py	(revision 221)
+++ podboy/gui.py	(working copy)
@@ -70,7 +70,7 @@
         self.list.size_hint_align_set(-1, -1)
         self.box.pack_end(self.list)
         self.list.show()
-        self.list.callback_clicked_add(self.on_list_item_clicked)
+        #self.list.callback_clicked_add(self.on_list_item_clicked)
         self.list_items_icons = None
 
         self.status = elementary.Label(self.main.win)
@@ -83,7 +83,7 @@
 
         self.box_actions = elementary.Box(self.main.win)
         self.box_actions.horizontal_set(True)
-        self.box_actions.homogenous_set(True)
+        self.box_actions.homogeneous_set(True)
         self.box_actions.size_hint_weight_set(1, 0)
         self.box_actions.size_hint_align_set(-1, 0)
         self.box.pack_end(self.box_actions)
@@ -590,7 +590,7 @@
 
         box_actions = elementary.Box(iwin)
         box_actions.horizontal_set(True)
-        box_actions.homogenous_set(True)
+        box_actions.homogeneous_set(True)
         box_actions.size_hint_align_set(-1, -1)
         box.pack_end(box_actions)
         box_actions.show()
@@ -1225,7 +1225,7 @@
 
         box = elementary.Box(self.main.win)
         box.horizontal_set(True)
-        box.homogenous_set(False)
+        box.homogeneous_set(False)
         box.size_hint_weight_set(1, 1)
         box.size_hint_align_set(-1, -1)
         self.box.pack_end(box)
@@ -1234,7 +1234,7 @@
         self.icon_cover = elementary.Icon(self.main.win)
         self.icon_cover.size_hint_align_set(0, 0)
         self.icon_cover.prescale_set(self.main.win.size[0] / 4. / elementary.scale_get())
-        self.icon_cover.scale_set(0, 0)
+        self.icon_cover.scale_set(0)
         box.pack_end(self.icon_cover)
         self.icon_cover.callback_clicked_add(self.show_cover)
         self.icon_cover.show()
@@ -1284,7 +1284,7 @@
 
         box_actions = elementary.Box(self.main.win)
         box_actions.horizontal_set(True)
-        box_actions.homogenous_set(True)
+        box_actions.homogeneous_set(True)
         box_actions.size_hint_weight_set(1, 0)
         box_actions.size_hint_align_set(-1, -1)
         box_actions.show()
@@ -1294,7 +1294,7 @@
         icon_play.file_set(THEME_EDJ, "podboy/play")
         icon_play.show()
         self.btn_play = elementary.Button(self.main.win)
-        self.btn_play.icon_set(icon_play)
+        self.btn_play.content_set(icon_play)
         self.btn_play.size_hint_weight_set(1, 1)
         self.btn_play.size_hint_align_set(-1, -1)
         self.btn_play.callback_clicked_add(self.toggle_play)
@@ -1305,7 +1305,7 @@
         icon_stop.file_set(THEME_EDJ, "podboy/stop")
         icon_stop.show()
         self.btn_stop = elementary.Button(self.main.win)
-        self.btn_stop.icon_set(icon_stop)
+        self.btn_stop.content_set(icon_stop)
         self.btn_stop.size_hint_weight_set(1, 1)
         self.btn_stop.size_hint_align_set(-1, -1)
         self.btn_stop.callback_clicked_add(self.stop)
@@ -1489,7 +1489,7 @@
             self.gst.stop()
 
             self.slider_position.value = 0
-            self.btn_play.icon_get().file_set(THEME_EDJ, "podboy/play")
+            self.btn_play.content_get().file_set(THEME_EDJ, "podboy/play")
 
             self.episode.update_last_pos(self.last_pos)
 
@@ -1524,7 +1524,7 @@
             self.stop_gst_message_timer()
             self.gst.pause()
 
-            self.btn_play.icon_get().file_set(THEME_EDJ, "podboy/play")
+            self.btn_play.content_get().file_set(THEME_EDJ, "podboy/play")
         else:
             self.main.dbus.set_suspend_mode(False)
 
@@ -1539,7 +1539,7 @@
             self.start_slider_position_timer()
             self.start_gst_message_timer()
 
-            self.btn_play.icon_get().file_set(THEME_EDJ, "podboy/pause")
+            self.btn_play.content_get().file_set(THEME_EDJ, "podboy/pause")
 
 
 #
@@ -1609,8 +1609,8 @@
         toggle.text_set("Delete Played")
         toggle.size_hint_align_set(-1, 0)
         toggle.state_set(episodes_old_auto_delete)
-        toggle.text_part_set("on", "Yes")
-        toggle.text_part_set("off", "No")
+        toggle.part_text_set("on", "Yes")
+        toggle.part_text_set("off", "No")
         toggle.callback_changed_add(self.toggle_episodes_old_auto_delete)
         box2.pack_end(toggle)
         toggle.show()
@@ -1664,8 +1664,8 @@
         toggle.text_set("Audio Output")
         toggle.size_hint_align_set(-1, 0)
         toggle.state_set(db.get_setting('bluetooth') or False)
-        toggle.text_part_set("off", "No")
-        toggle.text_part_set("on", "Yes")
+        toggle.part_text_set("off", "No")
+        toggle.part_text_set("on", "Yes")
         toggle.callback_changed_add(self.toggle_bluetooth)
         box_bt.pack_end(toggle)
         toggle.show()
@@ -1725,8 +1725,8 @@
         toggle.text_set("Fullscreen")
         toggle.size_hint_align_set(-1, 0)
         toggle.state_set(self.fullscreen)
-        toggle.text_part_set("off", "No")                                       
-        toggle.text_part_set("on", "Yes")
+        toggle.part_text_set("off", "No")                                       
+        toggle.part_text_set("on", "Yes")
         toggle.callback_changed_add(self.toggle_fullscreen)
         box_display.pack_end(toggle)
         toggle.show()
@@ -1741,8 +1741,8 @@
         toggle.text_set("Orientation")
         toggle.size_hint_align_set(-1, 0)
         toggle.state_set(db.get_setting('display_orientation') or False)
-        toggle.text_part_set("off", "Portrait")                                       
-        toggle.text_part_set("on", "Landscape")
+        toggle.part_text_set("off", "Portrait")                                       
+        toggle.part_text_set("on", "Landscape")
         toggle.callback_changed_add(self.toggle_orientation)
         box_display.pack_end(toggle)
         toggle.show()
@@ -1798,8 +1798,8 @@
         toggle.style_set("toggle")
         toggle.size_hint_align_set(-1, 0)
         toggle.state_set(db.get_setting('gui_list_multi_select') or False)
-        toggle.text_part_set("off", "No")                                       
-        toggle.text_part_set("on", "Yes")
+        toggle.part_text_set("off", "No")                                       
+        toggle.part_text_set("on", "Yes")
         toggle.callback_changed_add(self.toggle_list_multi_select)
         box_interface.pack_end(toggle)
         toggle.show()
@@ -1844,7 +1844,7 @@
         box_actions.size_hint_weight_set(1, 0)
         box_actions.size_hint_align_set(-1, 0)
         box_actions.horizontal_set(True)
-        box_actions.homogenous_set(True)
+        box_actions.homogeneous_set(True)
         box.pack_end(box_actions)
         box_actions.show()
  
@@ -2010,13 +2010,13 @@
         self.obj.parent_set(self.main.win)
 
         self.box = elementary.Box(self.main.win)
-        self.obj.content_set("top", self.box)
+        self.obj.part_content_set("top", self.box)
 
         # middle button
         btn = elementary.Button(self.main.win)
         btn.text_set(label)
         btn.callback_clicked_add(self._close)
-        self.obj.content_set("middle", btn)
+        self.obj.part_content_set("middle", btn)
 
         # bouton
         btn = elementary.Button(self.main.win)
@@ -2111,7 +2111,7 @@
 
         box_actions = elementary.Box(self.win)
         box_actions.horizontal_set(True)
-        box_actions.homogenous_set(True)
+        box_actions.homogeneous_set(True)
         box_actions.size_hint_weight_set(1, 0)
         box_actions.size_hint_align_set(-1, -1)
         box.pack_end(box_actions)
