diff -uNr git.orig//config.mk git/config.mk
--- git.orig//config.mk	2011-04-06 00:09:37.876727863 +0200
+++ git/config.mk	2011-04-06 00:12:05.164727971 +0200
@@ -5,8 +5,8 @@
 PKGDATADIR = ${DATADIR}/ffphonelog
 
 VALAFLAGS = --pkg elm --pkg posix --pkg gio-2.0
-INCS = $(shell pkg-config --cflags elementary gio-2.0)
-LIBS = $(shell pkg-config --libs elementary gio-2.0)
+INCS = $(shell pkg-config --cflags elementary ecore evas gio-2.0)
+LIBS = $(shell pkg-config --libs elementary ecore evas gio-2.0)
 
 CPPFLAGS = -DPKGDATADIR=\"${PKGDATADIR}\"
 CC=cc
