From f0578c263e1dfd7565db620e4cd93f069274a9c9 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Mon, 30 Jan 2012 17:49:49 +0100
Subject: [PATCH 07/11] ui_roster: drop unused _item_grp_sel

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 ui_roster.c |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/ui_roster.c b/ui_roster.c
index 9b5454c..aae0436 100644
--- a/ui_roster.c
+++ b/ui_roster.c
@@ -144,10 +144,6 @@ static Eina_Bool _item_grp_state_get(const Roster_Item_Grp *item, Evas_Object *o
   return EINA_FALSE;
 }
 
-static void _item_grp_sel(const Roster_Item_Grp *item, Evas_Object *obj, void *event_info) {
-  DEBUG("Selected Group %s", item->grp);
-}
-
 static const Elm_Genlist_Item_Class _item_grp_class={
   .item_style  = "default",
   .func = {
-- 
1.7.8.4

