From 0ca0f53877886d2a8afc26858510cd12bcec13d9 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Thu, 1 Dec 2011 20:09:02 +0100
Subject: [PATCH] fix name of changed genlist/gengrid callbacks

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 ffphonelog.vala |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/ffphonelog.vala b/ffphonelog.vala
index a5e04db..27176ea 100644
--- a/ffphonelog.vala
+++ b/ffphonelog.vala
@@ -297,7 +297,7 @@ class CallsList
 	lst.horizontal_mode_set(ListMode.COMPRESS);
 	itc.item_style = "double_label";
 	// XXX could libeflvala be fixed to avoid these casts
-	itc.func.label_get = (GenlistItemLabelGetFunc) get_label;
+	itc.func.text_get = (GenlistItemLabelGetFunc) get_label;
 	itc.func.content_get = (GenlistItemContentGetFunc) get_content;
 	lst.smart_callback_add("expand,request", expand);
 	lst.smart_callback_add("contract,request", contract);
-- 
1.7.8.rc4

From e28998da9173054fe915bc7aa9156ff40c88e43c Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Thu, 1 Dec 2011 22:15:29 +0100
Subject: [PATCH] gen

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 ffphonelog.vala |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/ffphonelog.vala b/ffphonelog.vala
index 27176ea..b8450f3 100644
--- a/ffphonelog.vala
+++ b/ffphonelog.vala
@@ -297,7 +297,7 @@ class CallsList
 	lst.horizontal_mode_set(ListMode.COMPRESS);
 	itc.item_style = "double_label";
 	// XXX could libeflvala be fixed to avoid these casts
-	itc.func.text_get = (GenlistItemLabelGetFunc) get_label;
+	itc.func.text_get = (GenlistItemTextGetFunc) get_text;
 	itc.func.content_get = (GenlistItemContentGetFunc) get_content;
 	lst.smart_callback_add("expand,request", expand);
 	lst.smart_callback_add("contract,request", contract);
@@ -432,7 +432,7 @@ class CallsList
 			    GenlistItemFlags.NONE, null);
     }
 
-    static string get_label(void *data, Elm.Object? obj, string part)
+    static string get_text(void *data, Elm.Object? obj, string part)
     {
 	return ((CallItem) data).get_label(part);
     }
-- 
1.7.8.rc4

