=== modified file 'lib/waudio/Makefile'
--- lib/waudio/Makefile	2013-01-26 02:26:38 +0000
+++ lib/waudio/Makefile	2013-02-24 17:06:00 +0000
@@ -33,7 +33,7 @@
 CFLAGS = -fPIC -DHAVE_ANDROID_OS=1 $(CONFIG_DEFINES)
 
 # modify linker command-line options
-LDFLAGS += --sysroot=$(rootfs_path) $(PKGLDFLAGS) -L ../libmedia/
+LDFLAGS += $(PKGLDFLAGS) -L ../libmedia/
 
 SHARED_LIBS += -lmedia -lpthread -lrt
 

