From 1ad4042d780f5a65466b012965d3fb1fbceb0bee Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Sun, 12 Jan 2014 16:30:59 +0100
Subject: [PATCH] ui-utils: Replace deprecated elm_notify_orient_set

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/util/ui-utils.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/util/ui-utils.c b/src/util/ui-utils.c
index 10f24db..da0dc82 100644
--- a/src/util/ui-utils.c
+++ b/src/util/ui-utils.c
@@ -572,7 +572,7 @@ ui_utils_notify(Evas_Object *parent, const char *label, int timeout)
 	Evas_Object *notify, *bx, *bt, *lb;
 	notify = elm_notify_add(parent);
 	evas_object_size_hint_weight_set(notify, EVAS_HINT_EXPAND, EVAS_HINT_EXPAND);
-	elm_notify_orient_set(notify, ELM_NOTIFY_ORIENT_CENTER);
+	elm_notify_align_set(notify, 0.5, 0.5);
 	elm_notify_timeout_set(notify, timeout);
 
 	bx = elm_box_add(parent);
-- 
1.8.5.2

