From b6bb7a0c988f21edd876d338a681dc8bd7209ef8 Mon Sep 17 00:00:00 2001
From: Christophe Chapuis <chris.chapuis@gmail.com>
Date: Tue, 14 May 2024 21:57:00 +0000
Subject: [PATCH] lk2nd_update_device_tree: improve lk2nd self-test

Upstream-Status: Pending

Signed-off-by: Christophe Chapuis <chris.chapuis@gmail.com>
---
 app/aboot/lk2nd-device.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app/aboot/lk2nd-device.c b/app/aboot/lk2nd-device.c
index 588dc17c..160b5ca3 100644
--- a/app/aboot/lk2nd-device.c
+++ b/app/aboot/lk2nd-device.c
@@ -459,7 +459,7 @@ static void lk2nd_update_panel_compatible(void *fdt)
 void lk2nd_update_device_tree(void *fdt, const char *cmdline)
 {
 	/* Don't touch lk2nd/downstream dtb */
-	if (strstr(cmdline, "lk2nd"))
+	if (strstr(cmdline, "lk2nd") == cmdline)
 		return;
 
 	if (lk2nd_dev.panel.compatible)
