Upstream-Status: Pending
---
diff --git a/arch/arm64/boot/dts/qcom/sdm450-xiaomi-rosy.dts b/arch/arm64/boot/dts/qcom/sdm450-xiaomi-rosy.dts
index 74aec8001c4c..e4f7b6375600 100644
--- a/arch/arm64/boot/dts/qcom/sdm450-xiaomi-rosy.dts
+++ b/arch/arm64/boot/dts/qcom/sdm450-xiaomi-rosy.dts
@@ -110,6 +110,6 @@ &ft5406_ts {
 
 &panel {
-	compatible = "xiaomi,rosy-panel";
+	compatible = "xiaomi,rosy-ft8613-csot\0xiaomi,rosy-panel";
 };
 
 &pm8953_l23 {
