From f801088c97be5e114e055f7725fc2401963d3046 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Mon, 2 May 2011 10:43:12 +0200
Subject: [PATCH 1/4] e-wm-config-illume2-shr: drop conf_imc, was merged with
 conf_intl in r59054

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 e.src |    6 ------
 1 files changed, 0 insertions(+), 6 deletions(-)

diff --git a/e.src b/e.src
index 6093c8a..6559e56 100644
--- a/e.src
+++ b/e.src
@@ -85,12 +85,6 @@ group "E_Config" struct {
 			value "priority" int: 0;
 		}
 		group "E_Config_Module" struct {
-			value "name" string: "conf_imc";
-			value "enabled" uchar: 1;
-			value "delayed" uchar: 1;
-			value "priority" int: 0;
-		}
-		group "E_Config_Module" struct {
 			value "name" string: "conf_keybindings";
 			value "enabled" uchar: 1;
 			value "delayed" uchar: 1;
-- 
1.7.5.rc3

